### PR TITLE
adjust HPU warmup: use dummy inputs with shape more close to real scenario 

### PR DESCRIPTION
**Pull request by kaixuanliu**
In original implementation, we use dummy inputs with shapes like [1,128], [1,256],[2,128],[2,256] to do warmup, aiming to generate recipe cache in warmup stage. And in real serving scenario, we padding the input_ids/attention_masks to shapes we cached in warmup stage. However, we found precision issue for reranker models following [tei docs](https://github.com/huggingface/text-embeddings-inference?tab=readme-ov-file#using-re-rankers-models) . We think it may be because we used wrong graphs during replay stage. Hence we adjust the `create_warmup_batch` function in this PR, to make the dummy inputs more close to real scenario, hence during warmup stage, these inputs will also be padded in python backend and will generate right recipe caching/graph, which will be the same with serving stage. We made several round experiments w/ this PR, and the wrong output issue disappears after this PR.

### CODE REVIEW

**Comment by kaixuanliu** [comment]
@regisss , pls help review, thx!

**Review by regisss — APPROVED** [review]
LGTM

**Comment by regisss** [comment]
cc @Narsil 
The methods `warmup_hpu` and `create_warmup_batch` are only used on HPU
